crates/bevy_app/src/app_builder.rs

```diff
@@ -273,13 +427,44 @@ impl AppBuilder {
         self
     }
 
+    /// Adds a group of plugins
+    ///
+    /// Bevy plugins can be grouped into a set of plugins. By default
+    /// bevy provides a few lists of plugins that can be used to kickstart
+    /// the development.
+    ///
+    /// Current plugins offered are [`DefaultPlugins`] and [`MinimalPlugins`].
+    ///
+    /// ## Example
+    /// ```
+    /// # use bevy_app::prelude::*;
+    /// #
+    /// App::build();
+    ///     //.add_plugins(MinimalPlugins)
```

Comment:
```suggestion
    /// ## Example
    /// ```
    /// # use bevy_app::{prelude::*, PluginGroupBuilder};
    /// #
    /// # // Dummy created to avoid using bevy_internal, which pulls in to many dependencies.
    /// # struct MinimalPlugins;
    /// # impl PluginGroup for MinimalPlugins {
    /// #     fn build(&mut self, group: &mut PluginGroupBuilder){;}
    /// # }
    /// #
    /// App::build()
    ///     .add_plugins(MinimalPlugins);
```
By creating a Dummy implementation, you can have the example look like actual Bevy Code.